docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,134 @@
+
+You can [set up](/docs/access-controls/guides/dual-authz/#dual-authorization-1) dual authorization by applying two [role templates](/docs/access-controls/guides/role-templates/):
+
+- [The reviewer](#the-reviewer)
+- [The reviewee](#the-reviewee)
+
+### The reviewer
+You can enable some users to review other users' role escalation requests by applying a role template similar to the following:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: <role name>
+spec:
+  allow:
+    review_requests:
+      roles: ["role-one", "role-two", "role-three"]
+```
+
+Assign `spec.allow.review_requests.roles` to a list of role names. When a user with the role in `metadata.name` requests access to one of the roles listed in `spec.allow.review_requests.roles`, your Teleport access plugins notify reviewers of the request and relay their response to your Teleport cluster.
+
+### The reviewee
+You can require a user to request access from reviewers by applying a role template similar to the following:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: <role name>
+spec:
+  allow:
+    request:
+      roles: ["role-one", "role-two", "role-three"]
+      thresholds:
+        - approve: 2
+          deny: 1
+```
+
+The `spec.allow.request.roles` field lists the names of other roles that a user with this role can request. When such a user [requests access](/docs/enterprise/workflow/#access-requests-setup) to one of these roles, Teleport requests a review via your access plugins. The `spec.allow.requests.roles.thresholds` field indicates how many reviews are required to approve or deny the request.
+
+
+## Automatically prevent some roles from requesting others
+In theory, a malicious Teleport user could request a more privileged role and trick a reviewer into granting access. You can prevent such a scenario by defining role templates that prohibit users from even requesting access to particular roles.
+
+The `spec.deny` field has the same possible properties as the `spec.allow` field we described [earlier](#require-multiple-reviews-for-role-requests) except, rather than enabling actions, this field disables them. For example, the `spec.deny.requests.roles` field is a list of roles that a user is prohibited from requesting.
+
+As an illustration, we have assigned user `myuser` to the `user` role, which we defined using the following template:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: user
+spec:
+  deny:
+    request:
+      roles: ['admin']
+
+```
+
+Next, `myuser` attempts to request the `admin` role.
+
+```
```

Comment:
let's use code block here (described in docs/docs)

Reply:
👍